tasks/local_git_users.yml

```diff
@@ -1,18 +1,21 @@
 ---
 - name: Identify gitea users
-  ansible.builtin.command: su - {{ gitea_user }} -c '{{ gitea_full_executable_path }} -c {{ gitea_configuration_path }}/gitea.ini admin user list'
+  ansible.builtin.command:
+    cmd: "{{ gitea_full_executable_path }} -c {{ gitea_configuration_path }}/gitea.ini admin user list"
   become: true
+  become_user: "{{ gitea_user }}"
   register: _giteausers
   changed_when: false
 
 - name: Use gitea cli to create user
   become: true
-  ansible.builtin.command: |
-    su - {{ gitea_user }} -c \
-    '{{ gitea_full_executable_path }} -c {{ gitea_configuration_path }}/gitea.ini \
-    admin user create --username "{{ user.name }}" \
-    --password "{{ user.password }}" --email "{{ user.email }}" \
-    --must-change-password={{ user.must_change_password }} --admin={{ user.admin }}'
+  become_user: "{{ gitea_user }}"
```

Comment:
if you run this role as non-root-user, I had a lot problems with the become_user method.

But as far as I know with the https://docs.ansible.com/ansible/latest/collections/community/general/sudosu_become.html ``community.general.sudosu`` that should be fixed.

Could you add a ``become_method: community.general.sudosu``?

Reply:
That should be easy to test in the CI.
My ansible user is non-root currently but not sure if I changed it after for this host/role/play.
It seems strange to me become or su not making the job and sudosu seems stranger to me. like people doing `sudo su -` and not `sudo -s`.

Reply:
Well, I tested it and still got the message:

```
TASK [l3d.git.gitea : Identify gitea users] **************
...
Escalation requires password
fatal: [git.example.org]: FAILED! => {
    "msg": "Missing sudo password"
}
```

But when I use sudosu instead of sudo it works...

```
  become: true
  become_user: "{{ gitea_user }}"
  become_method: community.general.sudosu
```

But I think I could set the become_method outside the role and then it should work for me too...
Let me test that...

Reply:
Well, that works. It is approved.